metals/src/main/scala/scala/meta/internal/metals/testProvider/TestSuitesProvider.scala

```diff
@@ -84,12 +130,30 @@ final class TestSuitesProvider(
           )
         )
 
+      val deleted = mutable.ArrayBuffer[(BuildTarget, RemoveTestSuite)]()
```

Comment:
Since this is a tuple all the accessing we do with it is a bit confusing with the `._1` and `._2` ceremony. For easiness of reading, is it possible to just store this as a `TestEntry` as well? I also think it's nicer to keep it in one format right until we send it out, then convert it to an `Event`. So internally we are only dealing with entries. wdyt?